scripts/publish.sh

```diff
@@ -0,0 +1,208 @@
+
+# Inputs:
+# release/
+#	tag
+#	version
+#	body
+#	commit_sha
+#	<assets> (cf-*, L*, R*)
+# src-ci/
+
+TAG=$(cat release/tag)
+VER=$(cat release/version)
+NOW=$(date -u --iso-8601=s | sed -e 's/\+.*$/Z/')
+
+PR_BRANCH=cf-plugin-backup-${TAG}
+PR_TITLE="Release cf-plugin-backup ${TAG}"
+PR_DESC="Release cf-plugin-backup ${TAG} from https://github.com/SUSE/cf-plugin-backup/releases/${TAG}"
+
+echo 'TAG:    ' ${TAG}
+echo 'VERSION:' ${VER}
+echo 'NOW:    ' ${NOW}
+echo 'BRANCH: ' ${PR_BRANCH}
+echo 'gh_USER:' ${GITHUB_USER}
+# No, we will not show the token.
+
+# # ## ### ##### ######## ############# #####################
+## Assemble index information
+
+echo
+echo Assembling the yaml ...
+
+rm -f  index.yml
+cat >> index.yml <<EOF
+- authors:
+  - contact:  ${CONTACT}
+    homepage: ${HOMEURL}
+    name:     ${NAME}
+  binaries:
+EOF
+
+platform() {
+    local path="$1"
+    case $path in
+	*darwin-amd64*)  echo osx ;;
+	*linux-386*)     echo linux32 ;;
+	*linux-amd64*)   echo linux64 ;;
+	*windows-386*)   echo win32 ;;
+	*windows-amd64*) echo win64 ;;
+	*)               echo Unable to determine platform code
+	                 exit 1 ;;
+    esac
+}
+
+# Ignore the checksum files
+rm release/*.SHA256
+
+for asset in release/cf-plugin-backup-*
+do
+    echo
+    echo Processing $asset ...
+    HASH=$(sha1sum $asset | sed -e 's/ .*$//')
+    BASE=$(basename $asset)
+    PLAT=$(platform $asset)
+    echo '- SHA1:' $HASH
+    echo '- PLAT:' $PLAT
+
+    cat >> index.yml <<EOF
+  - checksum: ${HASH}
+    platform: ${PLAT}
+    url: https://github.com/SUSE/cf-plugin-backup/releases/download/${TAG}/${BASE}
+EOF
+done
+
+cat >> index.yml <<EOF
+  company:     ${COMPANY}
+  created:     ${NOW}
+  description: A plugin that enables backup and restore of the CCDB via the CF API 
+  homepage:    https://github.com/SUSE/cf-plugin-backup
+  name:        cf-plugin-backup
+  updated:     ${NOW}
+  version:     ${VER}
+EOF
+
+echo
+echo Assembled index:
+cat index.yml
+echo
+
+# # ## ### ##### ######## ############# #####################
+## Start on making the upstream PR
+
+echo
+echo Creating the pull request ...
+
+# Get the "hub" cli -- https://hub.github.com/hub.1.html
+curl -L ${HUB_RELEASE} | tar xvz --wildcards --strip-components=2 '*/bin/hub'
+HUB="${PWD}/hub"
+chmod +x ${HUB}
+
+# Clone the upstream github repo. The chosen directory name is
+# independent of the configured upstream name, during testing we may
+# operate on a repo with a different name.
+#
+# Note also that the chosen directory matches what the GO code expects
+# for its imports to work.
+
+echo ; echo = Clone
+
+mkdir -p                 src/code.cloudfoundry.org
+${HUB} clone ${UPSTREAM} src/code.cloudfoundry.org/cli-plugin-repo
+
+# Enter local checkout
+cd src/code.cloudfoundry.org/cli-plugin-repo
+
+# Admin stuff for the checkout. Notably, point it to the SUSE fork,
+# that is where we will push the changes to.
+
+${HUB} config user.email "cf-ci-bot@suse.de"
```

Comment:
I think this should be configurable as well. We already define it in the concourse secrets.